Makefile

```
@@ -153,6 +153,18 @@ test:
 	@echo -e "\n[gotest] \033[1;33mgo test -v\033[0m $(GO_TEST_DIRS)\033[0m"
 	@$(BUILD_FLAGS) go test -v $(GO_TEST_DIRS)
 
+coverage:
+	@echo -e "[Ginkgo] \033[1;33mgo test -coverprofile=gotestginkgo.out -v\033[0m $(TEST_DIRS)\033[0m"
+	@$(BUILD_FLAGS) go test -coverprofile=gotestginkgo.out -v $(TEST_DIRS) -ginkgo.show-node-events
+	@echo -e "[Ginkgo] \033[1;33mgo tool cover -html=gotestginkgo.out -o gotestginkgo.html\033[0m"
+	@$(BUILD_FLAGS) go tool cover -html=gotestginkgo.out -o gotestginkgo.html
+
+	@echo -e "\n[gotest] \033[1;33mgo test -v\033[0m $(GO_TEST_DIRS)\033[0m"
+	@$(BUILD_FLAGS) go test -coverprofile=gotest.out -v $(GO_TEST_DIRS)
+	@echo -e "[Ginkgo] \033[1;33mgo tool cover -html=gotest.out -o gotest.html\033[0m"
```

```suggestion
	@echo -e "[gotest] \033[1;33mgo tool cover -html=gotest.out -o gotest.html\033[0m"
```
I assume you want `gotest` label here.